x-pack/plugin/eql/src/test/resources/test_string_functions.toml

```diff
@@ -411,73 +411,3 @@ description = "Test the `substring` function when the case already matches"
     [[substring.fold.tests]]
     expression = '''substring("hello world", -5, -1)'''
     expected = "worl"
-
-[wildcard]
```

Comment:
should this one be replaced with `like`? is this file even used?

Reply:
Yes, file still in use.